config/config.go

```diff
@@ -36,7 +36,10 @@ type Config struct {
 	// Token represent the configuration to generate N-token to connect to athenz.
 	Token Token `yaml:"ntoken"`
 
-	// Role represent the configuration to generate role token from athenz server.
+	// Access represent the configuration to retrieve access token from athenz server.
+	Access Access `yaml:"access_token"`
```

Comment:
like `roletoken`, maybe `accesstoken` is better?
I think you follow the `Role` naming convention but it is not a good naming convention. Maybe we using `AccessToken` for now and refactor the current implementation like `Role` -> `RoleToken`? 
```suggestion
	AccessToken AccessToken `yaml:"accesstoken"`
```

Reply:
agreed on `roletoken` and `access-token`.

for config name convention, this issue applies.
https://github.com/yahoojapan/athenz-client-sidecar/issues/34